spacy_llm/tasks/entity_linking.py

```diff
@@ -0,0 +1,310 @@
+import csv
+import re
+import warnings
+from pathlib import Path
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Type, Union
+
+import jinja2
+import spacy
+from pydantic import BaseModel
+from spacy.language import Language
+from spacy.pipeline import EntityLinker
+from spacy.scorer import Scorer
+from spacy.tokens import Doc, Span
+from spacy.training import Example
+
+from ..registry import registry
+from ..ty import CandidateSelector, ExamplesConfigType
+from .templates import read_template
+from .util import SerializableTask
+
+_DEFAULT_EL_TEMPLATE_V1 = read_template("entity_linking")
```

Comment:
Rename template to `.v1`

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/937c04d90e2e4f4181d7476f9c2ad8676b052e0e.